browser.js

```diff
@@ -39,19 +39,29 @@ class PDFMerger {
   }
 
   async _getInputFile (inputFile) {
-    return new Promise((resolve) => {
-      if (inputFile instanceof Buffer) {
-        resolve(inputFile)
-      } else {
-        const fileReader = new window.FileReader()
-
-        fileReader.onload = function (evt) {
-          resolve(fileReader.result)
-        }
+    if (inputFile instanceof Buffer || inputFile instanceof ArrayBuffer) {
+      return inputFile
+    }
+    if (typeof inputFile === 'string' || inputFile instanceof String) {
+      const res = await window.fetch(inputFile)
+      const ab = await res.arrayBuffer()
+      return ab
+    }
+    if (inputFile instanceof window.File) {
+      const fileReader = new window.FileReader()
 
-        fileReader.readAsArrayBuffer(inputFile)
+      fileReader.onload = function (evt) {
+        return fileReader.result
```

Comment:
I _think_ this PR broke the `FileReader` implementation, at least from looking at the code. This return doesn't do anything, because it's an anonymous function. The outer `_getInputFile` never gets the result and I think it will reach the `throw` at the bottom?